src/transformers/models/mra/cuda_kernel.cu

```diff
@@ -0,0 +1,383 @@
+#include "cuda_kernel.h"
```

Comment:
Can you move the CUDA kernels in the new `kernels` folder? We keep them all here. (You may need to rebase)